Source/Server/.dolittle/bounded-context.json

```diff
@@ -14,10 +14,14 @@
         "embeddings": {
             "production": "MongoDB",
             "development": "MongoDB"
+        },
+        "readModels": {
+            "development": "n/a",
+            "development": "n/a"
```

Comment:
Ahh - this might be why it works. Change this to look more like it does in Production.

Reply:
Also, as codacy says - one of these should be production.

Reply:
Done